Sources/TuistLoader/Loaders/TemplateLoader.swift

```diff
@@ -9,8 +9,15 @@ public protocol TemplateLoading {
     /// Load `TuistScaffold.Template` at given `path`
     /// - Parameters:
     ///     - path: Path of template manifest file `name_of_template.swift`
+    ///     - plugins: List of available plugins.
     /// - Returns: Loaded `TuistScaffold.Template`
-    func loadTemplate(at path: AbsolutePath) throws -> TuistGraph.Template
+    func loadTemplate(at path: AbsolutePath, plugins: Plugins) throws -> TuistGraph.Template
+}
+
+extension TemplateLoading {
```

Comment:
Let's not over-use extensions. The implementation of this function can be in the body of `TemplateLoader`.

Reply:
I've added such extension to guarantee retrocompatibility to whoever was using that public method as-it-was and avoid a breaking change - `TemplateLoading` is a public protocol.

Reply:
`TuistLoader` is not part of the public interface (which is just `ProjectDescription` and `ProjectAutomation`, so no problema in changing it

Reply:
Understood!